deploy/helm/nfd-operator/templates/rbac.yaml

```
@@ -1,17 +1,23 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    {{- include "node-feature-discovery-operator.labels" . | nindent 4 }}
+  name: {{ include "node-feature-discovery-operator.name" . }}-service-account
```

Again, shouldn't we use fullname here? Also the `-service-account` feels redundant here

```suggestion
  name: {{ include "node-feature-discovery-operator.fullname" . }}
```

ditto other `name:`s